website/data-storage/README.md

```diff
@@ -1 +1,14 @@
 # @gear-js/data-storage
+
+### Database migration
+
+```bash
+# generate migration
+$ yarn migration:generate src/database/migrations/<MIGRATION_NAME>
```

Comment:
Is there any migration naming convention?

Reply:
Simple answer the `question`, yes.

Depend on the convention between developers in `team`. 
Usually, migration has two basic parts:
1. migration number or id (That part will generate automatically with TypeORM).
2. the second part is the name of migration. The name should tell us what should do a new migration. Some people write the second part with `camelcase` or `snakecase`.